platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/fontscolors/FlatLafDark-highlights.xml

```diff
@@ -49,4 +49,8 @@
     <fontcolor name="text-limit-line-color" foreColor="ff555555"/>
     <fontcolor name="trailing-whitespace"/>
     <fontcolor name="west-sidebars-color"/>
+    <fontcolor name="coverage-covered" bgColor="005000"/>
+    <fontcolor name="coverage-uncovered" bgColor="404040"/>
+    <fontcolor name="coverage-inferred" bgColor="1c501c"/>
```

Comment:
What does "inferred" mean? This green color is almost identical to that of nbcoverage-covered.

Reply:
Well, that's a good question. Tried to look it up on jacoco, found nothing. Maybe did not try hard enough. I have not seen any sample where it is came up.

Reply:
OK, they seem to be really close in the existing defaults as well. Maybe they're supposed to look the same...

Reply:
Honestly I do not know. I'd leave it for the "community test-pad" If someone find an incurred and the coloring disturbed, let's change it until then, it is Ok. Unless I find an incurred line first...

Reply:
Sure.